charts/sentry/values.yaml

```diff
@@ -1797,111 +1797,6 @@ zookeeper:
 # See https://github.com/bitnami/charts/tree/master/bitnami/kafka
 kafka:
   enabled: true
-  provisioning:
-    enabled: true
-    # Topic list is based on files below.
-    # - https://github.com/getsentry/snuba/blob/master/snuba/utils/streams/topics.py
-    # - https://github.com/getsentry/self-hosted/blob/master/install/create-kafka-topics.sh#L6
-    # Note that snuba component might fail if you set `hooks.snubaInit.kafka.enabled` to `false` and remove the topics from this default topic list.
-    topics:
-      - name: events
-        config:
-          "message.timestamp.type": LogAppendTime
-      - name: event-replacements
-      - name: snuba-commit-log
-        config:
-          "cleanup.policy": "compact,delete"
-          "min.compaction.lag.ms": "3600000"
```

Comment:
Could you please add these custom settings to the place where it should be now, or at least add topic customisation guide to the documentation?

Reply:
Hi @NikolaiIuzhaninYS, :wave: 

I'm not sure if this is possible. Sentry now automatically provisions Kafka topics and has also removed Kafka topic creation :
https://github.com/getsentry/self-hosted/commit/a6cb07691018bbe66360208b5f5f507f64b3f606

The cleanup policy can sometimes break the snuba api when you have too many events in Kafka.

Reply:
So it's not even possible to set partition counts ?

There used to be a env variable called `TOPIC_PARTITION_COUNTS`, I'm not sure it still work.


@TartanLeGrand what's your status on the PR ?

Reply:
`TOPIC_PARTITION_COUNTS` for snuba conf only. I didn’t find a way to configure the number of partitions in the sentry code

I think we can accept this change as is and later implement an alternative way to create topics

Reply:
I've a question why the init db job are not a hook anymore @TartanLeGrand ?

Reply:
Because they launch only if all health rules are ok. The change in the last version of Bitnami Nginx broke this thing. For have the health rules ok on the Nginx we need to have migrate and init before. 😅

Reply:
It could be a good idea to add an issue to the nginx bitnami helmchart. I don't know if it's a normal behaviour.

Reply:
It's a normal behavior 😉

Reply:
I think we then need an initСontainer in each sentry and snuba pod, which will check that the migrations have been applied and only then launch the primary container(s)

Reply:
Instead a job ? 🤔

Reply:
Nope. initContainer's task is to check that all migrations have been applied by the job and only in this case launch the container with the new version of sentry. This will be an ideal replacement for helm hooks